scripts/zones/Oldton_Movalpolos/mobs/Goblin_Wolfman.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+-- Area: Oldton Movalpolos (11)
+--  Mob: Goblin Wolfman
+-----------------------------------
+
```

Comment:
There's a mixin for adding 2HR to mob

```mixins = {require("scripts/mixins/job_special")};```

Also, this script references both FORCE_SPAWN_QM_RESET_TIME and SCRAWLED_WRITING, so it should require the files that define them.

```
require("scripts/zones/Oldton_Movalpolos/MobIDs");
require("scripts/globals/settings");
```